packages/wrangler/src/containers/index.ts

```diff
@@ -26,35 +30,41 @@ export const containers = (
 			"build [PATH]",
 			"build a dockerfile",
 			(args) => buildYargs(args),
-			(args) => handleFailure(buildCommand)(args)
+			(args) => handleFailure(buildCommand, containersScope)(args)
 		)
 		.command(
 			"push [TAG]",
 			"push a tagged image to a Cloudflare managed registry, which is automatically integrated with your account",
 			(args) => pushYargs(args),
-			(args) => handleFailure(pushCommand)(args)
+			(args) => handleFailure(pushCommand, containersScope)(args)
+		)
+		.command(
+			"apply",
+			"apply the changes in the container applications to deploy",
+			(args) => applyCommandOptionalYargs(args),
+			(args) => handleFailure(applyCommand, containersScope)(args)
```

Comment:
Why is this PR adding a new command?

Reply:
ah I think that's from when I using an in-progress branch as the base for these changes, I'll remove that